tests/smoke_tests/test_cluster_job.py

```diff
@@ -845,14 +845,15 @@ def test_add_pod_annotations_for_autodown_with_launch():
             # Autodown is set.
             f'sky launch -y -c {name} -i 10 --down --num-nodes 2 --cpus=1 --cloud kubernetes',
             # Get names of the pods containing cluster name.
-            f'pod_1=$(kubectl get pods -o name | grep {name} | sed -n 1p)',
-            f'pod_2=$(kubectl get pods -o name | grep {name} | sed -n 2p)',
+            f'pod_1=$(kubectl get pods -o name | grep {name} | sed -n 1p) && '
             # Describe the first pod and check for annotations.
-            'kubectl describe pod $pod_1 | grep -q skypilot.co/autodown',
-            'kubectl describe pod $pod_1 | grep -q skypilot.co/idle_minutes_to_autostop',
+            'pod_tag=$(kubectl describe $pod_1); echo "$pod_tag"; echo "$pod_tag" | grep -q skypilot.co/autodown && '
+            'pod_tag=$(kubectl describe $pod_1); echo "$pod_tag"; echo "$pod_tag" | grep -q skypilot.co/idle_minutes_to_autostop',
+            # Get names of the pods containing cluster name.
+            f'pod_2=$(kubectl get pods -o name | grep {name} | sed -n 2p) && '
             # Describe the second pod and check for annotations.
-            'kubectl describe pod $pod_2 | grep -q skypilot.co/autodown',
```

Comment:
I assume this test had been working because this line would evaluate to `kubectl describe pod`, which would list all pods any way 😅 Thanks for fixing!

Reply:
yep, the issue came up when the k8s has multiple smoke tests running : )